code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -49,7 +49,7 @@
 
 /obj/item/weapon/pinpointer/examine(mob/user)
 	..(user)
-	for(var/obj/machinery/nuclearbomb/bomb in world)
+	for(var/obj/machinery/nuclearbomb/bomb in machines)

```

Comment:
I'll make `machines` be the general machines list, and an `SSmachinery.processing_machines` list for the processing ones.